desenvolvimento-front-end/bloco-11-introducao-a-react/dia-01-hello-world-no-react/pratica/src/Content.jsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+
+const conteudos = [
+  {
+    conteudo: 'High Order Functions',
+    bloco: 8,
+    status: 'Aprendido'
+  },
+  {
+    conteudo: 'Composicao de Componentes',
+    bloco: 11,
+    status: 'Aprendendo',
+  },
+  {
+    conteudo: 'Composicao de Estados',
+    bloco: 12,
+    status: 'Aprenderei'
+  },
+  {
+    conteudo: 'Redux',
+    bloco: 16,
+    status: 'Aprenderei'
+  },
+];
+
+const contents = (object) => {
+  return (<ul className='card'>
```

Comment:
No exercício pedia pra adicionar um atributo key com o conteúdo de cada card, você acabou esquecendo de colocar. 
O enunciado deixou meio vago o que era pra ser feito, mas basicamente era isso que ele queria: 
```suggestion
  return (<ul key={object.conteudo} className='card'>
```

Reply:
Gostei bastante que tu fez uma função pra criar cada ul e jogar no content. Ficou bem legível separando dessa forma parabéns 😸

Reply:
> No exercício pedia pra adicionar um atributo key com o conteúdo de cada card, você acabou esquecendo de colocar. O enunciado deixou meio vago o que era pra ser feito, mas basicamente era isso que ele queria:

O enunciado eu vou fazer depois, em todos, e deixar em um READ-ME, é uma tarefa a se fazer kk, mas agradeço por lembrar, e realmente não lembrei dessa parte da Key, obrigado